releases/index.md

```diff
@@ -240,7 +257,7 @@ The following binaries are not suitable for production environments:
                     {% if r.no_source == "true" %}
                 N/A
                     {% else %}
-                <code>cockroachdb/cockroach{% if r.version contains "-" %}-unstable{% endif %}:{{ r.version }}</code>
+                <b>Intel{% if r.docker_arm == "true" %}/ ARM{% endif %}</b>: <code>cockroachdb/cockroach{% if r.version contains "-" %}-unstable{% endif %}:{{ r.version }}</code>
```

Comment:
```suggestion
                <b>Intel{% if r.docker_arm == "true" %}/ARM{% endif %}</b>: <code>{{ r.docker_image }}:{{ r.version }}</code>
```

We populate the `docker_image` variable in `releases.csv`. I must have forgotten to include it the first time.